src/code42cli/cmds/search/extraction.py

```diff
@@ -61,12 +69,22 @@ def handle_response(response):
                 events = _get_alert_details(sdk, events)
             except Exception as ex:
                 handlers.handle_error(ex)
+
         handlers.TOTAL_EVENTS += len(events)
-        event = None
-        for event in events:
-            output_logger.info(event)
-        if event:
-            last_event_timestamp = extractor._get_timestamp_from_item(event)
+        if len(events):
+            format_header = (
+                {key: key.capitalize() for key in events[0].keys()}
+                if display_all
+                else {
+                    key: key.capitalize()
```

Comment:
This whole part is a bit confusing to follow.

Reply:
I don't really get the different between `display_all` and not `display_all` as far as this logic goes... maybe it'd be best to make some old-fashioned `if, else` statements here.

Reply:
I understand the pain in reading it, as this is the only way ternary operator is defined in python. 

Based on the `display_all` it initialises `format_header` variable. 
Otherwise I have to do either of below and former involves extra process and later involves extra declaration!! 
```
format_header =  {key: key.capitalize() for key in events[0].keys()}
if display_all:
    format_header = {key: key.capitalize() for key in events[0].keys() if...}
```

or

```
format_header = []
if display_all:
    format_header =  {key: key.capitalize() for key in events[0].keys()}
else:
   format_header =  {key: key.capitalize() for key in events[0].keys() if.....}
```